### DISCUSSION TITLE
Rewrite to API in Python running on AWS

### DISCUSSION THREAD

**Discussion by neosavvy** [vercel/community#972] [Help] [answered]
I have a backend Python API running at a location like https://api.dev.somethingoranother.com:30055

I have my rewrite in next.config.js like this:

``` async rewrites() {
    return [
      {
        source: '/api/backend/int/:path*',
        destination: 'https://api.dev.somethingoranother.com:30055/:path*',
      },
    ]
  },
```

When I cURL to https://api.dev.somethingoranother.com:30055/internal/whatever with POST I get the normal 200 response I expect. When I go through my rewrite like https://onboarding.somethingoranother.com/api/backend/int/internal/whatever I see a 307 redirect on my Python server and then nothing else happens until the Function log times out. 

This seems really janky and I'm lacking the tools to understand what's not working on the NextJS side. I'm currently deploying to vercel and this worked locally. I'd welcome any thoughts on what else to try as I'm at a loss as to what else I can do to debug this.

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @neosavvy! If you can share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), I would be happy to take a closer look. 

Otherwise, I would suggest double checking your [rewrite config](https://nextjs.org/docs/api-reference/next.config.js/rewrites), Project [domain settings](https://vercel.com/docs/concepts/projects/domains), [environment variables](https://vercel.com/docs/concepts/projects/environment-variables), and any related settings for your backend Python API. I would also recommend using [logs](https://vercel.com/docs/concepts/deployments/logs) and/or [log drains](https://vercel.com/docs/integrations/log-drains-overview) to see more details about what is happening.
